crates/dbsp/src/operator/communication/exchange.rs

```diff
@@ -875,17 +876,18 @@ where
 /// for this worker in the current clock cycle.  The scheduler should use
 /// [`ExchangeReceiver::register_ready_callback`] to get notified when the
 /// operator becomes schedulable.
-pub struct ExchangeReceiver<T, L>
+pub struct ExchangeReceiver<IF, T, L>
```

Comment:
shouldn't there be some documentation about IF and init somewhere?

Reply:
done